iac/modules/agreements-service/provider.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.14.2"
```

Comment:
Could make this value a variable with a default in `deploy-all/variables.tf` and pass in to each module - save repeating it in each provider.tf?

Reply:
I agree - that would be better - feels like there are improvements that could be made in these duplicated files. If ok - I will merge as-is for now in order to try and get the resources deployed asap, and improve as a follow on task